### ISSUE TITLE
Calling `toil clean` throws a boto3 error with toil[wdl] install

### ISSUE THREAD

**Issue by davidlougheed** [DataBiosphere/toil#4314]
When I run `toil clean` with a fresh install of `toil[wdl]==5.8.0`, I get the following traceback:

```
Traceback (most recent call last):
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/bin/toil", line 8, in <module>
    sys.exit(main())
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/lib/python3.10/site-packages/toil/utils/toilMain.py", line 14, in main
    modules = loadModules()
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/lib/python3.10/site-packages/toil/utils/toilMain.py", line 54, in loadModules
    from toil.utils import toilLaunchCluster  # noqa
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/lib/python3.10/site-packages/toil/utils/toilLaunchCluster.py", line 22, in <module>
    from toil.lib.aws.utils import build_tag_dict_from_env
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/lib/python3.10/site-packages/toil/lib/aws/utils.py", line 34, in <module>
    from toil.lib.aws import session
  File "/Users/davidlougheed/git/bentoV2/repos/wes/env/lib/python3.10/site-packages/toil/lib/aws/session.py", line 35, in <module>
    import boto3
ModuleNotFoundError: No module named 'boto3'
```

I am not using AWS and thus I should not need to install `boto3`.

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/TOIL-1260)
┆Issue Number: TOIL-1260

**Comment by davidlougheed** [comment]
Note that when `boto3` is installed (manually), it throws for `boto` too. After both are installed, it can clean happily.

**Comment by DailyDreaming** [comment]
Thanks for reporting this @davidlougheed !  This should be fixed by #4329.  Please reopen if this did not fix the issue.

**Comment by DailyDreaming** [comment]
@davidlougheed Thanks for reporting this!  It should be fixed now and released as 5.9.2.
